orderbook/openapi.yml

```diff
@@ -246,8 +246,11 @@ components:
           description: "Amount of sellToken available for the settlement contract to spend on behalf of the owner. Null if API was unable to fetch balance."
           $ref: "#/components/schemas/TokenAmount"
           nullable: true
-        executedSellAmount:
-          description: "The total amount of sellToken that has been executed for this order."
+        executedNetSellAmount:
+          description: "The total amount of sellToken that has been executed for this order including fees."
+          $ref: "#/components/schemas/BigUint"
+        executedGrossSellAmount:
+          description: "The total amount of sellToken that has been executed for this order without fees."
```

Comment:
I'm in favour of keeping the field named `executedSellAmount` with the contents being `value - fee`, and using `executedGrossSellAmount` for same as is today (equivalent to contract; `value + fee`)

Reply:
You are just lazy to update the frontend 😉 (I guess backwards compatibility is a reasonable argument, although we are still early in our development process).
I really don't have a strong feeling, just felt like if there is a gross there should also be a net...

Reply:
Well, that is the case most of the time... :sweat_smile: but what I'm proposing will require changes since right now I'm [deducing `executedFeeAmount` from `executedSellAmount` to get the final value](https://github.com/gnosis/gp-ui/blob/develop/src/api/operator/utils.ts#L82).

My reasoning is to have the name and contents of `executed<Buy|Sell>Amount` equivalent, with the extra info for `sells` in another variable.
So the next dev working on this is not surprised that `executedBuyAmount` doesn't have the same "meaning" as `executedSellAmount`.
Although one could argue that the original meaning is correct already since it's what's added/removed from user's balance... :shrug:

Reply:
I don't mind too much about the name of this, but I have to say, gross/net confused me. 

Here the gross says `without fees`, which I would read as "not including fees". If they don't include fees, for me is they don't have the fees applied. But I think you mean the opposite. 

Maybe is easier just to say executedSellAmount, executedSellAmountBeforeFees

I'd be ok with net/gross, just the "without" in the description confused me.

Reply:
I think I mixed up net/gross in the first version. Might be a sign that using executedSellAmount vs executedSellAmountBeforeFees is more intuitive for non-accountants. Will change